### DISCUSSION TITLE
Error: Attempt to get an MFT record with an old reference

### DISCUSSION THREAD

**Discussion by Shad0wCell** [EricZimmerman/KapeFiles#554] [Q&A] [answered]
Hi all,

Upon utilising the most recent version of KAPE within a virtualised environment I am testing, I am recieving the following error on every file it is trying to read.

Error: Attempt to get an MFT record with an old reference

Has anyone come across this before? If so, do you know of it's root cause and remedy for this?

Kind Regards

**Comment by EricZimmerman** [comment, marked as answer]
it happens, but rarely

a reboot usually fixes it. as to WHY, i do not know

**Reply by Shad0wCell** [reply]
Excellent, I'll try that.

Thanks for the response :-)
